product_sequence/models/product_product.py

```diff
@@ -7,32 +7,23 @@
 
 
 class ProductProduct(models.Model):
-    _inherit = 'product.product'
+    _inherit = "product.product"
```

Comment:
Why this change is needed?

Reply:
@moylop260 i use black :-)

Reply:
IMHO you should disable black for this case.
Since that we don't have discussed about to use double quotes
However, you can start a new thread to propose define it:
 - https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

For now, if we don't have define it use double-quotes or single-quotes then other people could change from double to single and other people could change single to double.

The problem is that this kind of changes without previous definition makes big diffs and issues to merge backports or conflicts if there is 2 or more Pull Requests for same module.

Reply:
Could be but now there is no another PR, is not a big diff for this case.

Reply:
It is a big diff for this case
Since that the title talks about one change of depends but the diff has 11 additions and 15 deletions for manifest and 9 additions and 20 deletions for product_product.py where I expected just 1 or 2 lines changed.

It is valid if you have created first the rule to global guidelines because other people has different custom styles but not OCA ones.

There are developers using the following way
```python
    field_1          = fields.Datetime()
    field_name_large = fields.Integer()
    field_3          = fields.Char()
```

Then, start a war of big diff changing custom style 1 to custom style 2 (not OCA ones)

# Using Custom Style for each developer:

Commit1 Custom Style A - "Adding `required=True` to field1"
```diff
-    field_1          = fields.Datetime()
+    field_1 = fields.Datetime(required=True)
    field_name_large = fields.Integer()
-    field_3          = fields.Char()
+    field_3 = fields.Char()
```

Commit2 Custom Style B - "Adding readonly to field_name_large"
```diff
-    field_1 = fields.Datetime()
+    field_1          = fields.Datetime()
-    field_name_large = fields.Integer()
+    field_name_large = fields.Integer(readonly=True)
-    field_3 = fields.Char()
+    field_3          = fields.Char()
```

# Using all developers the same styles ruled by OCA guidelines:

Commit1 Style OCA - "Adding `required=True` to field1"
```diff
-    field_1 = fields.Datetime()
+    field_1 = fields.Datetime(required=True)
    field_name_large = fields.Integer()
    field_3 = fields.Char()
```

Commit2 Custom Style B - "Adding readonly to field_name_large"
```diff
    field_1 = fields.Datetime(required=True)
-    field_name_large = fields.Integer()
+    field_name_large = fields.Integer(readonly=True)
    field_3 = fields.Char()
```

Small diff without conflicts.

If you want create a new rule then you need first create a PR to guidelines document before.